.github/CODEOWNERS

```diff
@@ -36,3 +36,5 @@
 /src/command_modules/azure-cli-vm/ @adewaleo
 /src/command_modules/azure-cli-eventhubs/ @v-ajnava
 /src/command_modules/azure-cli-servicebus/ @v-ajnava
+/src/azure-cli/azure/cli/command_modules/apimgmt/ @zikalino
+/src/azure-cli/azure/cli/command_modules/apimgmt/ @zikalino
```

Comment:
Duplicate line (you likely ran create twice).

Reply:
indeed :-)